### DISCUSSION TITLE
rdiff-backup reports version as "DEV" on Mac M1 (Ventura)

### DISCUSSION THREAD

**Discussion by free-flyer** [Homebrew/discussions#4066] [Getting started]
I have installed rdiff-backup on my Mac Book Pro M1 (Ventura) using brew...

`brew install rdiff-backup`

When I check the version using...

`rdiff-backup -V`

it reports the version as "rdiff-backup DEV"

I believe the version should be 2.0.5, but how can I be sure of this when it reports the version as DEV ?

**Comment by cho-m** [comment]
This may be a side effect of manually moving the files into `libexec`. The `rdiff-backup` gets version number by looking up package info, which means it needs to be installed in location the corresponding Python executable can find.

I mentioned this in an open PR for new version (https://github.com/Homebrew/homebrew-core/pull/118506) so may explore ways to fix there.
